aggregator/rest_server/server.go

```diff
@@ -55,6 +55,7 @@ func (s *RestServer) Start() error {
 	s.logger.Info("Starting aggregator REST API.")
 
 	router := mux.NewRouter()
+	router.HandleFunc("/state-root", wrapRequest(s.listener.APIErrors, s.handleGetStateRootUpdateAggregation)).Methods("GET")
```

Comment:
Why do we need an additional endpoint while we already get this trhoguh state-root-update?

Reply:
`GetStateRootUpdateAggregation` returns too much, we don't need most of their information.

Reply:
https://github.com/Nuffle-Labs/nffl/pull/302/commits/39317dd6adf8aa7b2a8e5d576f00451065537ea7 & https://github.com/Nuffle-Labs/nffl/pull/302/commits/119506078ac821d02fac3db4637615b2a827162f